Projet/render/Moteur.h

```diff
@@ -303,6 +307,7 @@ template <class T, class TDevice> class CMoteur :public Singleton<T>
 
 	// La seule scène
 	std::shared_ptr<Scene> CurrentScene;
+	std::unique_ptr<Pitbull::Actor> SkyBox;
```

Comment:
Should the skybox be stored in the scene rather than the Engine ?

Reply:
i changed